### DISCUSSION TITLE
How to ignore source's unmapped fields?

### DISCUSSION THREAD

**Discussion by zwjohn** [nartc/mapper#529] [General]
Hi,
I am using AutoMapper in NestJS, and I noticed that unmapped fields in the nested object from the source are added to the destination's nested object. How do I ignore these unmapped fields?

Thanks!

**Comment by milosdakic** [comment]
When you say "unmapped fields in the nested object from the source are added to the destination's nested object" do you mean they are coming back as `undefined` or similar?
